sdk/communication/azure-communication-chat/tests/_shared/test_utils.py

```diff
@@ -0,0 +1,26 @@
+import unittest
```

Comment:
@annatisch where do you think we should put the tests for the shared code?

Reply:
I discussed this one with Anna offline before. Since the issue was raised in Chat, so we decide put it in Chat.